tools/deployment/package-helm/templates/database-statefulset.yaml

```diff
@@ -0,0 +1,133 @@
+apiVersion: "v1"
+kind: "Service"
+metadata:
+  name: {{ include "clp-package.fullname" . }}-database
+  labels:
+    {{- include "clp-package.labels" . | nindent 4 }}
+    app.kubernetes.io/component: "database"
+spec:
+  clusterIP: "None"
+  selector:
+    {{- include "clp-package.selectorLabels" . | nindent 4 }}
+    app.kubernetes.io/component: "database"
+  ports:
+    - port: 3306
+      targetPort: 3306
+---
+{{- if .Values.allowSbinScripts }}
+apiVersion: "v1"
+kind: "Service"
+metadata:
+  name: {{ include "clp-package.fullname" . }}-database-public
+  labels:
+    {{- include "clp-package.labels" . | nindent 4 }}
+    app.kubernetes.io/component: "database"
+spec:
+  type: "NodePort"
+  selector:
+    {{- include "clp-package.selectorLabels" . | nindent 4 }}
+    app.kubernetes.io/component: "database"
+  ports:
+    - port: 3306
+      nodePort: 30306
```

Comment:
shall we expose the port using NodePort, or ask users to run kubectl port-forward to expose ports explicitly for databases?